modules/aws-outposts-machine-set.adoc

```diff
@@ -0,0 +1,216 @@
+  -n openshift-machine-api -o yaml
+----
++
+--
+.Example output
+[source,yaml]
+----
+apiVersion: machine.openshift.io/v1beta1
+kind: MachineSet
+metadata:
+  labels:
+    machine.openshift.io/cluster-api-cluster: <infrastructure_id> # <1>
+  name: <infrastructure_id>-<role> # <2>
+  namespace: openshift-machine-api
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      machine.openshift.io/cluster-api-cluster: <infrastructure_id>
+      machine.openshift.io/cluster-api-machineset: <infrastructure_id>-<role>
+  template:
+    metadata:
+      labels:
+        machine.openshift.io/cluster-api-cluster: <infrastructure_id>
+        machine.openshift.io/cluster-api-machine-role: <role>
+        machine.openshift.io/cluster-api-machine-type: <role>
+        machine.openshift.io/cluster-api-machineset: <infrastructure_id>-<role>
+    spec:
+      providerSpec: # <3>
+# ...
+----
+<1> The cluster infrastructure ID.
+<2> A default node label. For AWS Outposts, you use the `outposts` role.
+<3> The omitted `<providerSpec>` section includes values that must be configured for your Outpost.
+--
+
+. Configure the new compute machine set to create edge compute machines in the Outpost by editing the following values in the `<new_machine_set_name_1>.yaml` file:
++
+--
+.Example compute machine set for AWS Outposts
+[source,yaml]
+----
+apiVersion: machine.openshift.io/v1beta1
+kind: MachineSet
+metadata:
+  labels:
+    machine.openshift.io/cluster-api-cluster: <infrastructure_id> # <1>
+  name: <infrastructure_id>-outposts-<availability_zone> # <2>
+  namespace: openshift-machine-api
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      machine.openshift.io/cluster-api-cluster: <infrastructure_id>
+      machine.openshift.io/cluster-api-machineset: <infrastructure_id>-outposts-<availability_zone>
+  template:
+    metadata:
+      labels:
+        machine.openshift.io/cluster-api-cluster: <infrastructure_id>
+        machine.openshift.io/cluster-api-machine-role: outposts
+        machine.openshift.io/cluster-api-machine-type: outposts
+        machine.openshift.io/cluster-api-machineset: <infrastructure_id>-outposts-<availability_zone>
+    spec:
+      metadata: {}
+      providerSpec:
+        value:
+          ami:
+            id: <ami_id> # <3>
+          apiVersion: machine.openshift.io/v1beta1
+          blockDevices:
+            - ebs:
+                volumeSize: 120
+                volumeType: gp2 # <4>
+          credentialsSecret:
+            name: aws-cloud-credentials
+          deviceIndex: 0
+          iamInstanceProfile:
+            id: <infrastructure_id>-outposts-profile
+          instanceType: m5.xlarge # <5>
+          kind: AWSMachineProviderConfig
+          placement:
+            availabilityZone: <availability_zone>
+            region: <region> # <6>
+          securityGroups:
+            - filters:
+              - name: tag:Name
+                values:
+                  - <infrastructure_id>-outposts-sg
```

Comment:
we are not instructing to create this security group, should it be better to leave the default?